src/redux/actions/embedding/loadEmbedding.js

```diff
@@ -5,10 +5,17 @@ const TIMEOUT_SECONDS = 90;
 
 const loadEmbedding = (experimentId, embeddingType) => async (dispatch, getState) => {
   // If a previous load was initiated, hold off on it until that one is executed.
-  if (getState().embeddings[embeddingType] && getState().embeddings[embeddingType].loading) {
+  if (getState().embeddings[embeddingType]?.loading
+    || getState().embeddings[embeddingType]?.data.length) {
     return null;
   }
 
+  const { pipeline } = getState().experimentSettings.pipelineStatus.status;
+
+  if (!pipeline?.startDate) {
+    return;
```

Comment:
I would stick to returning `null` (like the existing code), rather than `undefined`

Reply:
fixed in commit 953c3b9